client/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/kubernetes-csi/external-snapshotter/client/v2
+
+go 1.12
+
+require (
+	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d // indirect
+	golang.org/x/time v0.0.0-20200416051211-89c76fbcd5d1 // indirect
+	k8s.io/api v0.18.5
```

Comment:
Can you update this and all other k8s componentes to v0.19.0-rc.0?  We are in the process of updating all dependencies to 1.19.